api/src/services/listing.service.ts

```diff
@@ -329,6 +340,9 @@ export class ListingService implements OnModuleInit {
           in: listingIds.map((listing) => listing.id),
         },
       },
+      orderBy: {
```

Comment:
Design requested that we filter the list alphabetically by name instead of the current sort which is by ID

Reply:
Chatted w Em and updated to most recently published!